src/locales/en.json

```diff
@@ -22,13 +22,27 @@
       "WALKING": "Walking",
       "BUS": "Bus"
     },
+    "purposes": {
+      "HOME": "Home",
+      "WORK": "Work",
+      "SCHOOL": "School",
+      "SHOPPING": "Shopping",
+      "MEAL": "Meal",
+      "PERSONAL_MED": "Personal",
+      "EXERCISE": "Sport",
+      "ENTERTAINMENT": "Entertainment",
+      "PICK_DROP": "Pick & drop",
+      "OTHER_PURPOSE": "Unknown"
+    },
```

Comment:
nit: would be perfect if sorted alphabetically :)

Reply:
C'est trié en fonction des specs